bioblend/galaxy/tool_dependencies/__init__.py

```diff
@@ -0,0 +1,82 @@
+"""
+Contains possible interaction dealing with Galaxy dependency resolvers.
+"""
+from bioblend.galaxy.client import Client
+
+
+class ToolDependenciesClient(Client):
+
+    def __init__(self, galaxy_instance):
+        self.module = 'dependency_resolvers'
+        super().__init__(galaxy_instance)
+
+    def summarize_toolbox(self, index=None, tool_ids=None, resolver_type=None, include_containers=None, container_type=None, index_by=None):
+        """
+        GET /api/dependency_resolvers/toolbox
+
+        Summarize requirements across toolbox (for Tool Management grid). This is an experiemental
+        API particularly tied to the GUI - expect breaking changes until this notice is removed.
+
+        :type index: int
+        :param index: index of the dependency resolver
```

Comment:
Can you clarify a bit more what this means?

Reply:
It's the index of the dependency resolver with respect to the config file.  If you use the default config, index 0 is tool shed repositories, 1 is galaxy packages, 2 is conda with the right version etc.  The output is the dependency resolution with respect to the chosen resolver. It seems easier to document this as 'index of the dependency resolver' than try to accurately explain it - I'm not sure if what I just said about it is 100% correct.